ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -384,6 +388,17 @@ orderby submission.TimeSubmitted
             }
 
             FileStoragePrefix = FileManager.GetFileStoragePrefix(Event.ID, "");
+            if (!string.IsNullOrWhiteSpace(Puzzle.CustomCSSFile)) 
+            {
+                if (Puzzle.CustomCSSFile.StartsWith("$"))
+                {
+                    ContentFile content = await (from contentFile in context.ContentFiles
+                                                 where contentFile.Event == Event &&
+                                                 contentFile.ShortName == Puzzle.CustomCSSFile.Substring(1)
+                                                 select contentFile).SingleOrDefaultAsync();
+                    PossibleMaterialFile = content.Url.AbsoluteUri;

```

Comment:
The general approach makes sense but this will crash if no material is found - we should be gentler than that.

Reply:
Good catch, fixed!